packages/fxa-content-server/app/scripts/views/index.js

```diff
@@ -266,6 +266,14 @@ class IndexView extends FormView {
       .then(({ exists, hasPassword, hasLinkedAccount }) => {
         const nextEndpoint = exists ? 'signin' : 'signup';
 
+        // Temporary hack for React work that allows us to pass the entered `email` as
+        // a param. When 'signup' and 'signin' flows are both finished and we're ready,
+        // we can convert the index page over and pass this along with router state
+        // instead of param.
+        this.user.set({ emailFromIndex: email });
```

Comment:
Just curious if there weould be a difference between `this.user.set({ emailFromIndex: email });` and `this.user.set('emailFromIndex', email);`?

Reply:
I don't think there's a difference, so good catch on that being a little inconsistent. I'm guessing the former is usually for updating multiple values and the latter is for a single value, so I'll see if I can update all 3 with the first way.